### DISCUSSION TITLE
Next.js vs Redom.js

### DISCUSSION THREAD

**Discussion by aziyatali** [redom/redom#443] [Q&A] [answered]
I found out they have similar method of constructing the DOM elements. 
What are some differences and similarities you have noticed?

**Comment by pakastin** [comment, marked as answer]
RE:DOM just extends Web APIs and DOM. Node.js doesn't support DOM elements out of box at all and you need library for that.

Please refer to MDN for example https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction

**Reply by aziyatali** [reply]
Not Node.js but Next.js.
Autocomplete might have made typo.

**Reply by pakastin** [reply]
RE:DOM is tiny helpers around DOM, next.js is complex framework, so all different products
